src/System.Console/src/System/ConsoleColor.cs

```diff
@@ -8,6 +8,8 @@ namespace System
 
     public enum ConsoleColor
     {
+        Unknown = -1,
```

Comment:
Sorry I missed that comment. This is an interesting case because it actually never shows up in the IL as a reference because it is just a constant so technically it is compatible in that sense. However I agree it isn't something we really want folks to use directly so lets keep it only in the implementation for now and see how things go.